satellite/accounting/live/redis.go

```diff
@@ -119,7 +120,9 @@ func (cache *redisLiveAccounting) GetAllProjectTotals(ctx context.Context) (_ ma
 			if err != nil {
 				return Error.New("could not get total for project %s", id.String())
 			}
-			projects[*id] = int64(intval)
+			if !strings.HasSuffix(item.Key.String(), "bandwidth") {
```

Comment:
I would check this after verifying that they key isn't null or better just discarding the keys with length larger than the node ID, but I'm good to go with this to deploy it ASAP